profiles/car.lua

```diff
@@ -356,6 +356,7 @@ function process_way(profile, way, result, relations)
     -- routable. this includes things like status=impassable,
     -- toll=yes and oneway=reversible
     WayHandlers.blocked_ways,
+    WayHandlers.avoid_ways,
```

Comment:
Good catch. I think we never caught that since the default entries for `avoid` don't have speeds assigned.